lib/src/command/upgrade.dart

```diff
@@ -220,6 +225,18 @@ be direct 'dependencies' or 'dev_dependencies', following packages are not:
 
     _outputChangeSummary(changes);
 
+    // If any of the packages to upgrade are dependency overrides, then we
+    // show a warning.
+    final toUpgradeOverrides = toUpgrade
+        .where(entrypoint.root.dependencyOverrides.containsKey)
+        .toList();
+    if (toUpgradeOverrides.isNotEmpty) {
```

Comment:
```suggestion
    final hasUpgradeOverrides = toUpgrade
        .any(entrypoint.root.dependencyOverrides.containsKey);
    if (hasUpgradeOverrides) {
```

Reply:
Aren't you just making it less useful by now showing which packages are affected?

Reply:
Sorry this was wrong. Disregard